docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -36,7 +36,9 @@ Edit the `pom.xml` file to add the Quarkus gRPC extension dependency (just under
 
 Make sure you have `generate-code` goal of `quarkus-maven-plugin` enabled in your `pom.xml`.
 If you wish to generate code from different `proto` files for tests, also add the `generate-code-tests` goal.
-Please note that no additional task/goal is required for the Gradle plugin.
+Please note that no additional task/goal is required for the Gradle plugin. Note that we're adding a Jandex index via `jandex-maven-plugin`,
+so that changes to gRPC/proto definitions are picked up and dependencies are automatically scanned for beans.
```

Comment:
This text sounds like the Jandex plugin is required even if the gRPC classes are in the same jar as the application classes. This is not true.
The only situation in which it is required is when gRPC classes are in a dependency of the application. Then the dependency needs a Jandex index. 

I think this information would be best put in a separate section.

Reply:
Agreed that it sounds like it's required when it's not. I'll update stuff and put it in a separate section. Thanks

Reply:
fixed